packages/terra-image/src/Image.jsx

```diff
@@ -143,28 +143,35 @@ class Image extends React.Component {
       src, variant, isFluid, alt, placeholder, height, width, onLoad, onError, fit, ...customProps
     } = this.props;
 
-    const imageClasses = cx([
+    const imageClassAttributes = [
       'image',
       fit,
       variant,
       customProps.className,
       { fluid: isFluid },
-    ]);
+    ];
+
+    if (placeholder && this.state.isLoading) {
+      imageClassAttributes.push('hidden');
+    }
+
+    const imageClasses = cx(imageClassAttributes);
+
     delete customProps.className;
     if (!this.state.isLoading) {
       objectFitImages(this.ImageRef.current);
     }
     if (placeholder) {
       if (this.state.isLoading) {
         return (
-          <div>
-            <div className={cx('hidden')}>{this.createImage(customProps, imageClasses)}</div>
-            <div>{placeholder}</div>
-          </div>
+          <>
+            {this.createImage(customProps, imageClasses)}
+            {placeholder}
+          </>
         );
       }
 
-      return this.state.isError ? placeholder : this.createImage(customProps, imageClasses);
+      return this.state.isError ? placeholder : <>{this.createImage(customProps, imageClasses)}</>;
```

Comment:
Is the fragment here necessary?

Reply:
Nope, was just being safe with my initial fix. Just tested without it and works.